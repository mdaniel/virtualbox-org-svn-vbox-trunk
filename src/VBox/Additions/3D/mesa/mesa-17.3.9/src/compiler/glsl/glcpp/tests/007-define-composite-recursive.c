#define foo a bar
#define bar b baz
#define baz c foo
foo
bar
baz
