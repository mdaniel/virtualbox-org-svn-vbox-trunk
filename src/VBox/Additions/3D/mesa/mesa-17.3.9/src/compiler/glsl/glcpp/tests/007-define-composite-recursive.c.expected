


a b c foo
b c a bar
c a b baz
